src/index.ts

```diff
@@ -37,11 +37,24 @@ if (!globalThis.FormData) {
   }
 }
 
+interface RequestInitWithDuplex extends RequestInit {
+  /**
+   * `RequestInit#duplex` does not yet exist in the TS `lib.dom.d.ts` definition yet the native
+   * fetch implementation in Node 18+, `undici`, requires it for certain POST payloads.
+   *
+   * @see {@link https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1483}
```

Comment:
I created this issue this morning. The `RequestInit` types in [undici](https://npm.im/undici) have `RequestInit#duplex` defined[^1] but these types aren't in TS' core DOM type library so attempting to set `duplex` here to get these requests working on Node 18+, or for anyone using `undici`, would fail.

[^1]: https://github.com/nodejs/undici/blob/a3ce745b0c5d7c837ecae2359434ab4550a6e0ea/types/fetch.d.ts#L103-L118